lightly/api/api_workflow_client.py

```diff
@@ -118,7 +118,7 @@ def dataset_id(self) -> str:
         try:
             return self._dataset_id
         except AttributeError:
-            all_datasets: List[DatasetData] = self._datasets_api.get_datasets()
+            all_datasets: List[DatasetData] = self.get_datasets()
             datasets_sorted = sorted(all_datasets, key=lambda dataset: dataset.last_modified_at)
             last_modified_dataset = datasets_sorted[-1]
```

Comment:
this is kinda weird and cant imagine a situation where we want to auto-default to the the dataset that was last_modified_at?

Reply:
I will not remove this from the code although I understand your confusion.